docs/migration.md

```diff
@@ -0,0 +1,167 @@
+
+
+Migration from v1.2.8 to v2.0.0
+-------------------------------
+
+  
+
+This document is a guide to help you migrate from rdiff-backup v1.2.8
+(legacy version) to rdiff-backup v2.0.0 (new version).
+
+  
+
+This new version of rdiff-backup is compatible with repositories created
+with the legacy version. In other words, the data on the disk is fully
+compatible between the two. But the network protocol is not. For this
+reason, you might need to have a plan to transition from the legacy
+version to the new version depending of your use case.
+
+
+You have two options for your migration:
+
+1. Upgrade all instances of rdiff-backup to the new version. This option
+is recommended for **small deployment**. If this is your case, just
+follow the instructions <span
+style="font-weight: normal">[here](https://github.com/rdiff-backup/rdiff-backup#installation).

```

Comment:
Isn' the use of <span> here a bit excess? Also make "follow the instructions" the link itself and don't make any extra "here" a link, see why in https://uxmovement.com/content/why-your-links-should-never-say-click-here/